### PR TITLE
Update ethereum tests v11

### PR DESCRIPTION
**Pull request by ScottyPoi**
Re: #2038 Update ethereum-tests v11

* Points ethereum-tests submodule to latest version update (v11)
* Updates Difficulty tests in block package to use `ethereum-tests` instead of locally copied version
* Deletes locally copied version of `dfGrayGlacier/`

### CODE REVIEW

**Review by acolytec3 — DISMISSED** [review]
LGTM.  I added the testAllHardforks label and all those tests passed as well.  @holgerd77  @jochem-brouwer can we go ahead and merge this?

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "ethereum-tests"]
 	path = packages/ethereum-tests
 	url = https://github.com/ethereum/tests.git
-	branch = develop
+	branch = v11
```
Comment (jochem-brouwer) [inline .gitmodules:4]: Will this branch regularly get updated or should we revert back to the `develop` branch? (If that has the latest tests we should do so)

**Comment by jochem-brouwer** [comment]
One question otherwise we can merge!

File: .gitmodules (hunk 1 above)
Comment (acolytec3) [reply .gitmodules:4]: We should stay on `develop` methinks.  All the v11 tests have been merged in develop so as long as we pin the commit 95a8490f1f202dc9f58e910571d5d02082e0e641 that corresponds to the v11 release.

Comment (jochem-brouwer) [reply .gitmodules:4]: Just checked, `develop` has new commits and `v11` has not, so we should stay on develop

Comment (jochem-brouwer) [inline .gitmodules:4]:
```suggestion
	branch = develop
```

**Review by jochem-brouwer — APPROVED** [review]
LGTM

**Pull request merged**